src/python/T0Component/Tier0Feeder/Tier0FeederPoller.py

```diff
@@ -154,14 +161,30 @@ def algorithm(self, parameters = None):
             #
             # replays call data discovery only once (and ignore data status)
             #
+
+            # If no helper agents are given in the configuration, 
+            # this is a main agent that will ignore no streams
+            if not tier0Config.Global.HelperAgentStreams:
+                logging.info("No HelperAgent provided. This is a MainAgent. Processing all streams")
+                self.agentName = "MainAgent"
```

Comment:
If no HelperAgents are defined, we run with a MainAgent who will ignore no streams